src/Tribe/Views/V2/Assets.php

```diff
@@ -205,8 +205,12 @@ public function register() {
 				'conditionals' => [ $this, 'should_enqueue_frontend' ],
 				'groups'       => [ static::$group_key, static::$widget_group_key ],
 				'defer'        => true,
+				'in_footer'    => array( 'strategy'  => 'defer' ),
```

Comment:
Minor code format tweak:

```suggestion
				'in_footer'    => [ 'strategy' => 'defer' ],
```